docs/proposals/images/machine-health-check/mhc.plantuml

```diff
@@ -0,0 +1,20 @@
+@startuml
+start;
+:Machine Health Check controller;
+repeat
+  repeat
```

Comment:
This nested `repeat` block doesn't render as I think you probably want in the image. Is there some change you could make?

Reply:
mm I couldn't find a better way by looking here http://plantuml.com/guide